myMigrationScripts/migratePokemonLEvelAndMoves.js

```diff
@@ -0,0 +1,70 @@
+const connectDB = require('../config/db');
+const Pokemon = require('../models/Pokemons');
+const fetch = require('node-fetch');
+
+connectDB();
+
+// Get random number between minand max
+const getRandomBetween = (min, max) => {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+};
+
+// Get random moves
+const getRandomMoves = (moves, times) => {
+  // if there's less than 2 moves, just return them
+  if (moves.length <= times) return moves;
+  const result = [];
+  let randomIndex;
+  while (times > 0) {
+    randomIndex = getRandomBetween(0, moves.length - 1);
+    result.push(moves[randomIndex]);
+    moves.splice(randomIndex, 1);
+    times -= 1;
```

Comment:
It's generally a bad practice to modify something passed as an argument to the function. It works fine here, but in many cases (or languages) it might cause a bug.

Consider doing it like this:
```ts
const numOfRandomMoves = times;
while (numOfRandomMoves > 0) {
  ...
```
Same with `moves`, but it's less trivial, so let's keep it as it is and just learn more about shallow-copy and deep-copy: https://medium.com/javascript-in-plain-english/how-to-deep-copy-objects-and-arrays-in-javascript-7c911359b089 

I'd also consider renaming `times` to something more descriptive (like `numToRandomize`)